openshift.spec

```diff
@@ -3,14 +3,14 @@
 %global gopath      %{_datadir}/gocode
 %global import_path github.com/openshift/origin
 %{!?commit:
-%global commit 21fb40637c4e3507cca1fcab6c4d56b06950a149
+%global commit 7d5852ac026006bf6b29dafcd5388d78d4011403
 }
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 
 Name:           openshift
-Version:        0.2
+Version:        0.3
```

Comment:
Sorry git tag is v0.3,
~~~
$ git tag |grep v0.3
v0.3
~~~
so if it is changed to 0.3.0, it will mismatch between Version and Source0.
Please ignore my previous comment.